go/inst/instance_dao.go

```diff
@@ -204,6 +204,20 @@ func ReadTopologyInstance(instanceKey *InstanceKey) (*Instance, error) {
 	return ReadTopologyInstanceBufferable(instanceKey, false, nil)
 }
 
+// ReadTopologyInstances is a convenience method that calls ReadTopologyInstance
+// for all the instance keys and returns a slice of Instance.
+func ReadTopologyInstances(instanceKeys []InstanceKey) ([]*Instance, error) {
+	instances := make([]*Instance, 0)
+	for _, replicaKey := range instanceKeys {
+		replica, err := ReadTopologyInstance(&replicaKey)
+		if err != nil {
+			return nil, err
+		}
+		instances = append(instances, replica)
```

Comment:
we are reading instances, not replicas. Change suggested
```suggestion
	for _, instanceKey := range instanceKeys {
		instance, err := ReadTopologyInstance(&instanceKey)
		if err != nil {
			return nil, err
		}
		instances = append(instances, instance)
```